src/gameplaykit.cs

```diff
@@ -992,45 +1342,555 @@ interface GKMonteCarloStrategist : GKStrategist {
+
+		[Export ("initWithMaxNumberOfChildren:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (nuint maxNumberOfChildren);
+
+		[Export ("addElement:boundingRectMin:boundingRectMax:splitStrategy:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		void AddElement (ElementType element, Vector2 boundingRectMin, Vector2 boundingRectMax, GKRTreeSplitStrategy splitStrategy);
+
+		[Export ("removeElement:boundingRectMin:boundingRectMax:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		void RemoveElement (ElementType element, Vector2 boundingRectMin, Vector2 boundingRectMax);
+
+		[Export ("elementsInBoundingRectMin:rectMax:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		ElementType [] GetElements (Vector2 rectMin, Vector2 rectMax);
+	}
+
+	[iOS (10,0), TV (10,0), Mac (10,12, onlyOn64: true)]
+	[BaseType (typeof (GKComponent))]
+	interface GKSKNodeComponent : GKAgentDelegate {
+
+		[Static]
+		[Export ("componentWithNode:")]
+		GKSKNodeComponent FromNode (SKNode node);
+
+		[Export ("initWithNode:")]
+		IntPtr Constructor (SKNode node);
+
+		[Export ("node", ArgumentSemantic.Strong)]
+		SKNode Node { get; set; }
+	}
+
+	interface IGKSceneRootNodeType { }
+
+	[iOS (10,0), TV (10,0), Mac (10,12, onlyOn64: true)]
+	[Protocol]
+	interface GKSceneRootNodeType {
+	}
+
+	[iOS (10,0), TV (10,0), Mac (10,12, onlyOn64: true)]
 	[BaseType (typeof (NSObject))]
+	interface GKScene : NSCopying, NSCoding {
+
+		[Static]
+		[Export ("sceneWithFileNamed:")]
+		[return: NullAllowed]
+		GKScene FromFile (string filename);
+
+		[Export ("entities")]
+		GKEntity [] Entities { get; }
+
+		[NullAllowed, Export ("rootNode", ArgumentSemantic.Assign)]
+		IGKSceneRootNodeType RootNode { get; set; }
+
+		[Export ("graphs")]
+		NSDictionary<NSString, GKGraph> Graphs { get; }
+
+		[Export ("addEntity:")]
+		void AddEntity (GKEntity entity);
+
+		[Export ("removeEntity:")]
+		void RemoveEntity (GKEntity entity);
+
+		[Export ("addGraph:name:")]
+		void AddGraph (GKGraph graph, string name);
+
+		[Export ("removeGraph:")]
+		void RemoveGraph (string name);
+	}
+
+	[iOS (10,0), TV (10,0), Mac (10,12, onlyOn64: true)]
+	[Category]
+	[BaseType (typeof (SKNode))]
+	interface SKNode_GameplayKit {
+
+		// Inlined them in SKNode to avoid ugly syntax
+		//[Static]
+		//[Export ("obstaclesFromSpriteTextures:accuracy:")]
+		//GKPolygonObstacle [] ObstaclesFromSpriteTextures (SKNode [] sprites, float accuracy);
+
+		//[Static]
+		//[Export ("obstaclesFromNodeBounds:")]
+		//GKPolygonObstacle [] ObstaclesFromNodeBounds (SKNode [] nodes);
+
+		//[Static]
+		//[Export ("obstaclesFromNodePhysicsBodies:")]
+		//GKPolygonObstacle [] ObstaclesFromNodePhysicsBodies (SKNode [] nodes);
+
+		[Export ("entity")]
+		GKEntity GetEntity ();
+
+		[Export ("setEntity:")]
+		void SetEntity (GKEntity entity);
+	}
+
+	[iOS (9,1), TV (9,0), Mac (10,12, onlyOn64: true)]
```

Comment:
Let's mark them as `[iOS (10,0)] [TV(10,0)]` since the old version likely does not work (on new devices and/or with the new bindings).